node/actors/bft/src/leader/tests.rs

```diff
@@ -254,23 +271,25 @@ async fn replica_prepare_high_qc_of_future_view() {
     );
 }
 
-#[ignore = "fails/unsupported"]
 #[tokio::test]
-async fn replica_prepare_non_validator_signer() {
```

Comment:
how is this test different than replica_prepare_non_validator_signer?

Reply:
Similar to https://github.com/matter-labs/era-consensus/pull/44#discussion_r1414167099. 

Removed.